tests/tests/build.rs

```diff
@@ -6,67 +6,69 @@ use tempfile::tempdir;
 use crate::rojo_test::io_util::{get_working_dir_path, BUILD_TESTS_PATH, ROJO_PATH};
 
 macro_rules! gen_build_tests {
-    ( $($test_name: ident,)* ) => {
+    ( $($test_name:ident: $out_format:literal,)* ) => {
         $(
             paste::item! {
                 #[test]
                 fn [<build_ $test_name>]() {
                     let _ = env_logger::try_init();
 
-                    run_build_test(stringify!($test_name));
+                    run_build_test(stringify!($test_name), $out_format);
                 }
             }
         )*
     };
 }
 
 gen_build_tests! {
-    init_csv_with_children,
-    attributes,
-    client_in_folder,
-    client_init,
-    csv_bug_145,
-    csv_bug_147,
-    csv_in_folder,
-    deep_nesting,
-    gitkeep,
-    ignore_glob_inner,
-    ignore_glob_nested,
-    ignore_glob_spec,
-    infer_service_name,
-    infer_starter_player,
-    init_meta_class_name,
-    init_meta_properties,
-    init_with_children,
-    issue_546,
-    json_as_lua,
-    json_model_in_folder,
-    json_model_legacy_name,
-    module_in_folder,
-    module_init,
-    optional,
-    project_composed_default,
-    project_composed_file,
-    project_root_name,
-    rbxm_in_folder,
-    rbxmx_in_folder,
-    rbxmx_ref,
-    script_meta_disabled,
-    server_in_folder,
-    server_init,
-    txt,
-    txt_in_folder,
-    unresolved_values,
-    weldconstraint,
+    init_csv_with_children: "rbxmx",
+    attributes: "rbxmx",
+    client_in_folder: "rbxmx",
+    client_init: "rbxmx",
+    csv_bug_145: "rbxmx",
+    csv_bug_147: "rbxmx",
+    csv_in_folder: "rbxmx",
+    deep_nesting: "rbxmx",
+    gitkeep: "rbxmx",
+    ignore_glob_inner: "rbxmx",
+    ignore_glob_nested: "rbxmx",
+    ignore_glob_spec: "rbxmx",
+    infer_service_name: "rbxlx",
+    infer_starter_player: "rbxlx",
+    init_meta_class_name: "rbxmx",
+    init_meta_properties: "rbxmx",
+    init_with_children: "rbxmx",
+    issue_546: "rbxmx",
+    json_as_lua: "rbxmx",
+    json_model_in_folder: "rbxmx",
+    json_model_legacy_name: "rbxmx",
+    module_in_folder: "rbxmx",
+    module_init: "rbxmx",
+    optional: "rbxmx",
+    project_composed_default: "rbxmx",
+    project_composed_file: "rbxmx",
+    project_root_name: "rbxmx",
+    rbxm_in_folder: "rbxmx",
+    rbxmx_in_folder: "rbxmx",
+    rbxmx_ref: "rbxmx",
+    script_meta_disabled: "rbxmx",
+    server_in_folder: "rbxmx",
+    server_init: "rbxmx",
+    txt: "rbxmx",
+    txt_in_folder: "rbxmx",
+    unresolved_values: "rbxlx",
+    weldconstraint: "rbxmx",
```

Comment:
Do we use binary formats in tests at all, or do the snaps for that suck too much to use here? Either way, I think this is enough here as long as we do test that binary works elsewhere.

Reply:
We don't use them because they're not human readable. Realistically, this is all we need to ensure the competency of the `build` command since there's much better test coverage in rbx-dom to ensure parity between the binary and xml format.